src/main/java/org/threeten/extra/Interval.java

```diff
@@ -180,80 +186,94 @@ public static Interval parse(CharSequence text) {
         Objects.requireNonNull(text, "text");
         for (int i = 0; i < text.length(); i++) {
             if (text.charAt(i) == '/') {
-                return parseSplit(text.subSequence(0, i), text.subSequence(i + 1, text.length()));
+                CharSequence startStr = text.subSequence(0, i);
+                CharSequence endStr = text.subSequence(i + 1, text.length());
+                if (startStr.charAt(0) == 'P' || startStr.charAt(0) == 'p') {
+                    // duration followed by temporal
+                    TemporalAmount duration = parseDuration(startStr);
+                    Temporal temporal = parseTemporal(endStr);
+                    return Interval.of(Instant.from(minus(temporal, duration)), Instant.from(temporal));
```

Comment:
I wonder if situations like this should "saturate" the instant at Instant.MIN/MAX, to simplify the lives of callers

Reply:
I don't think that's a good idea. First, `Instant` itself doesn't saturate. Both of the following throw `java.time.DateTimeException: Instant exceeds minimum or maximum instant`.

```java
Instant.MIN.minusNanos(1);
```

```java
Instant.MAX.plusNanos(1);
```

Same for `OffsetDateTime.MIN` / `OffsetDateTime.MAX`, different exception.

What's more, `LocalDateRange` (which is somewhat similar to `Interval`) doesn't saturate either. The following ends up with `java.time.DateTimeException: Invalid value for Year (valid values -999999999 - 999999999): 1000000000`.

```java
LocalDateRange.of(LocalDate.MAX.minusDays(2), Period.ofDays(3));
```

In my opinion, any kind of "saturation" would be very surprising to callers.
Let's preserve the status quo.